### DISCUSSION TITLE
referanslar niye görünmüyor

### DISCUSSION THREAD

**Discussion by suhedaalkas** [IBAX-io/go-ibax#1045] [Soru Cevap - Türkçe] [answered]
referanslar niye görünmüyor bilgisi olan var mı

**Comment by Yusufknk** [comment, marked as answer]
Tüm puanların ve referansların görünmesi için sayfayı ingilizce yapın. İkinci bir husus referans ödüllerinin yansıması için her iki tarafın görevleri eksiksiz olarak yapması aynı zamanda sosyal medya kanallarını takipten çıkarmamaları gerekir.

**Comment by hes91** [comment]
Sistem bazen buga girebiliyor.Çıkıp tekrardan girmeyi dene.Tarayıcıyı kapa yeniden aç.
